### ISSUE TITLE
Ganger Chieftain

### ISSUE THREAD

**Issue by Pooparoop** [keyteki/keyteki#384]

**Comment by cryogen** [comment]
It's not a bug with the card, it's working as intended, but the UX could be better..

**Comment by romanogit** [comment]
Documenting my thoughts:
There are two different behaviors for optional targetting today:
- Optional on card ability
- Optional on a target

I believe every optional with a target should use target's optional (second optional above). We could force it through code or test cases.

The first optional is the "may" type, for example: "Play: You may exalt Dino-Fiend. If you do, deal 3D to a creature. ". I think this one should be using "may" prompt instead of clicking on the card.

Basically, this would require remove optional from the ability itself, which, imho, makes sense. 

Another major refactor :)
